llvm/lib/Transforms/Instrumentation/RealtimeSanitizer.cpp

```diff
@@ -45,6 +45,26 @@ static void insertCallAtAllFunctionExitPoints(Function &Fn,
         insertCallBeforeInstruction(Fn, I, InsertFnName);
 }
 
+static PreservedAnalyses rtsanPreservedAnalyses() {
+  PreservedAnalyses PA;
+  PA.preserveSet<CFGAnalyses>();
+  return PA;
+}
+
+static void transformRealtimeUnsafeFunction(Function &F) {
```

Comment:
Nit (take it or leave it):

Transform is a little vague, could be more descriptive as to the actual transformation taking place

Reply:
Thanks, agreed